screenpy/actions/make_note.py

```diff
@@ -54,9 +55,9 @@ def as_(self: SelfMakeNote, key: str) -> SelfMakeNote:
 
     def describe(self: SelfMakeNote) -> str:
         """Describe the Action in present tense."""
-        return f"Make a note under {self.key}."
+        return f"Make a note under {represent_prop(self.key)}."
```

Comment:
This could be `key_to_log` instead, just in case we change our mind on `represent_prop`.

Reply:
(And it looks like that's the approach you have in the other `describe` methods, to use `*_to_log`.)

Reply:
I've always worried that setting the variable in the constructor would somehow lead to a problem if anyone were to alter the attribute being logged after instantiation.  It might be a non-issue but it's one of the reasons I try to avoid it as much as possible. 

We want the value coming out of `describe` to be accurate at the moment it is called.

Reply:
I forgot that part of the reason we have `*_to_log` is for the `beat` message.  I kinda want to go through each of the classes and make those attributes properties.

Reply:
I think they will not need to be around any longer, ideally, once we figure out this whole update to the logging bit. Or maybe the description/beat itself becomes a property, i think that would be slick.

Either way, let's do the describe reimagining before we do a change like that, because it might not be necessary in that brave new world.

Reply:
>  I think that would be slick.

It really would.  the only reason I didn't earlier is because the pattern wasn't already being used.  I'll open an issue to add those at some point.